src/importexport/mei/internal/meireader.cpp

```diff
@@ -0,0 +1,78 @@
+/*
+ * SPDX-License-Identifier: GPL-3.0-only
+ * MuseScore-CLA-applies
+ *
+ * MuseScore
+ * Music Composition & Notation
+ *
+ * Copyright (C) 2021 MuseScore BVBA and others
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+#include <QFileInfo>
+#include <QMessageBox>
+
+#include "meireader.h"
+
+#include "meiimporter.h"
+
+#include "libmscore/masterscore.h"
+#include "engraving/engravingerrors.h"
+
+using namespace mu::iex::mei;
+using namespace mu::engraving;
+
+//---------------------------------------------------------
+//   MeiImportErrorDialog
+//---------------------------------------------------------
+
+/**
+ Show a dialog displaying the MEI import error(s).
+ */
+
+static int meiImportErrorDialog(QString text, QString detailedText)
+{
+    QMessageBox errorDialog;
```

Comment:
Please don't use QMessageBox, use `INJECT(framework::IInteractive, interactive)` service

Reply:
Sure, I can change this. The `QMessageBox` has a text but also a detailed text with a list of messages set with `QMessageBox::setDetailedText`.

<img width="517" alt="image" src="https://github.com/musescore/MuseScore/assets/689412/de3c7e89-6aae-494a-912a-943ff13d420a">

Could you point me to an example that does something similar with the `interactive` service?

Reply:
see 
```
IInteractive::Result Interactive::warning(const std::string& title, const Text& text, const std::string& detailedText,
                                          const ButtonDatas& buttons, int defBtn,
                                          const Options& options) const
```

Reply:
Thanks. It it now looking like this:

<img width="524" alt="image" src="https://github.com/musescore/MuseScore/assets/689412/ed024a4d-81a0-46f1-9936-28116d26b572">